src/cirrus/build.py

```diff
@@ -45,6 +67,13 @@ def main():
         'venv',
         'bin',
         'virtualenv')
+
+    # remove existing virtual env if building clean
+    if opts.clean is True and os.path.exists(venv_path):
+        cmd = "rm -r {0}".format(venv_path)
```

Comment:
Woops, that's wrong. Lesson: Do not review code at 10:30 at night. You may indeed want the f option.